### DISCUSSION TITLE
Plausible relies on Clickhosue supporting hyperscan for "path" based Goals

### DISCUSSION THREAD

**Discussion by StyxOfDynamite** [plausible/analytics#2342] [Anything Else]
This is definitely not a bug in Plausible, so I don't expect you to be able to fix it but felt it worth noting somewhere.

If you created a "path" based goal, Plausible queries Clickhouse using `multiMatchAllIndices`

This as far as I can tell from Clickhouse's docs, utilises intel's `hyperscan` which is not supported on M1 macs, so if you're running plausible locally with the supplied docker configuration, you'll get a 500 error from the `conversion` api

**Comment by ruslandoga** [comment]
👋

You can try using clickhouse `22.9`, it switched to `vectorscan` in that version.

Please see https://github.com/plausible/hosting/pull/57 for more info.

**Reply by ruslandoga** [reply]
I'll open a PR to update Makefile commands in this repo.

**Reply by ruslandoga** [reply]
https://github.com/plausible/analytics/pull/2343
